Makefile

```diff
@@ -91,18 +88,20 @@ unit-test: check-all-generated-files-up-to-date unit-test-no-generate
 
 .PHONY: unit-test-no-generate ## Run unit test only
 unit-test-no-generate:
+	go install github.com/cloudflare/cfssl/cmd/...@latest
 	CGO_ENABLED=0 go test  -tags=release ./pkg/... ./cmd/... $(UNIT_TEST_ARGS)
 
 .PHONY: unit-test-race
 unit-test-race: ## Run unit test with race detection
+	go install github.com/cloudflare/cfssl/cmd/...@latest
 	CGO_ENABLED=1 go test -race ./pkg/... ./cmd/... $(UNIT_TEST_ARGS)
```

Comment:
is `cfssl` binary being called inside the unit tests?

Reply:
unfortunately, yes https://github.com/eksctl-io/eksctl/blob/83e5cccfb245d3f281fc031605f8082af3e64ced/pkg/iam/oidc/testdata/Makefile#L15 

the unit tests are not "pure" unit tests - for example, they reach out to ECR to pull images